connectors/spark/src/main/scala/io/unitycatalog/spark/TableProps.java

```diff
@@ -0,0 +1,20 @@
+package io.unitycatalog.spark;
+
+import java.util.Map;
+
+public class TableProps {
+  private TableProps() {
+  }
+
+  public static final String FIXED_CREDENTIAL_ENABLED = "fixedCredential.enabled";
```

Comment:
"fixed" does not make intuitive sense to me. can we use "refresh" or "renew"?

Reply:
Done ! renamed it to `renewCredential.enabled`.